protocol/parser/parser.go

```diff
@@ -254,21 +270,26 @@ func parse(rpcInput RPCInput, blockParser spectypes.BlockParser, dataSource int)
 }
 
 type ParsedInput struct {
-	parsedBlock  int64
-	parsedHashes []string
+	parsedDataRaw string
+	parsedBlock   int64
+	parsedHashes  []string
 }
 
 func NewParsedInput() *ParsedInput {
 	return &ParsedInput{
-		parsedBlock:  spectypes.NOT_APPLICABLE,
```

Comment:
keep the default not applicable, it's a health habit

Reply:
Fixed in b971e49be1a56fa62241b86af335e1c9362bc33f

Reply:
you did a removel: [-] parsedBlock:  spectypes.NOT_APPLICABLE i commented to keep it